README.rst

```diff
@@ -199,6 +199,28 @@ For details of the JDBC driver options refer to the official documentation.
 
 NOTE: Option names and values are case-sensitive. The option value is specified as a character string.
 
+Specify the Query Results
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to specify where athena stores the `txt` and `txt.metadata` files containing the result of each query, you can specify it as follows:
```

Comment:
https://docs.aws.amazon.com/athena/latest/ug/querying.html
> A results file stored automatically in a CSV format (*.csv), and
> An Athena metadata file (*.csv.metadata). 

Is it txt? Or is it csv?

Reply:
you know, that's also partially right. sometimes we had txt and sometimes we had csv. maybe it depends on the query...